zipline/data/us_equity_pricing.py

```diff
@@ -111,13 +118,17 @@ def check_uint32_safe(value, colname):
         )
 
 
-def winsorise_uint32(df, column, *columns):
+@expect_element(invalid_data_behavior={'warn', 'raise', 'ignore'})
+@preprocess(columns=expect_non_empty)
+def winsorise_uint32(df, invalid_data_behavior, *columns):
     """Drops any record where a value would not fit into a uint32.
 
     Parameters
     ----------
     df : pd.DataFrame
         The dataframe to winsorise.
+    invalid_data_behavior : {'warn', 'raise', 'ignore'}
```

Comment:
idk how I feel about that because it doesn't tell me what the error is. This is not an argument you will be passing often so I think it is okay to be a very clear here.